R/RcppExports.R

```diff
@@ -75,8 +75,40 @@ sys_fonts <- function() {
     .Call('gdtools_sys_fonts', PACKAGE = 'gdtools')
 }
 
-best_family_match <- function(font_family = "sans") {
-    .Call('gdtools_best_family_match', PACKAGE = 'gdtools', font_family)
+#' Find best family match with fontconfig
+#'
+#' \code{match_family()} returns the best font family match for the
+#' fontconfig pattern constructed from the \code{bold} and
+#' \code{italic} arguments. The default pattern is bold italic to make
+#' sure the matched font has enough features to be used in R graphics
+#' (plain, bold, italic, bold italic). \code{match_font()} returns the
+#' font file from the best family match, along with some metada in the
+#' attributes.
+#'
+#' Fontconfig matching is controlled via the \code{fonts.conf}
```

Comment:
Could we make this an argument?  i.e. have `debug = TRUE` to automatically set the env var, and reset after the function call.